src/app/pages/debrief/components/safety-questions-card/safety-questions-card.html

```diff
@@ -0,0 +1,28 @@
+<ion-card id="vehicle-checks" *ngIf="data.length > 0; let questions">
```

Comment:
Can we update to `data?.length`, I've seen a few Sentry errors for similar instances before that we have recently solved.